src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -4555,6 +4555,9 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                         className="plots-oql-status-banner"
                         store={this.props.store}
                         tabReflectsOql={false}
+                        isUnaffected={
```

Comment:
@adamabeshouse what is the difference between tabReflectsOql and isUnaffected props?

Reply:
the naming is not great. It's for the use case of toggling oql filtering in a tab. If isUnaffected, then the tab is not affected by oql no matter what. If !isUnaffected then tabReflectsOql just means that the tab is currently not affected by oql. This difference is only meaningful if onToggle is set, meaning that the oql filtering can be toggled and there'll be a little button to do so.